ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml

```diff
@@ -14,6 +14,31 @@ tests:
   commands: make lint
   container:
     from: src
+
+- as: e2e-aws-ran-profile
+  optional: true
```

Comment:
If you want it to run on all PRs and block if it fails, probably need to remove `optional`